Base/usr/share/man/man1/drain.md

```diff
@@ -0,0 +1,38 @@
+## Name
+
+drain - Print file to stdout, while progressively deleting read segments
+
+## Synopsis
+
+```**sh
+$ drain [file]
+```
+
+## Description
+
+drain is a utility for simultaneously reading and deleting a file. It is
```

Comment:
Looking at other man pages, all references to `drain` in the Description and Warning sections should be surround with backticks to make it monospace.

Reply:
Oops merged this PR by mistake - feel free to address the backticks separately, my bad